src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -65,14 +68,23 @@ private static void setupDrivers(Application application) {
 
 		driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
 		DriverFeature.addDriver("Special line Simulator", driver);
+		
+		application.addTest("Custom Command", action -> CommandsFeature.getDriverCommandManager()
+				.getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver()));
+
 		DriverFeature.updateDriverInfo();
 	}
 
 	private static void setupWindows(Application application) {
+		
+		Reader reader = new SimpleFormatReader();
```

Comment:
Please, move this line just before:
CommandImportWindow commandImportWindow = new CommandImportWindow(CommandsFeature.getDriverCommandManager(), reader);